fuse_publishers/src/pose_2d_publisher.cpp

```diff
@@ -190,7 +188,10 @@ void Pose2DPublisher::onStart()
   // Start the tf timer
   if (publish_to_tf_)
   {
-    tf_publish_timer_.start();
+    tf_publish_timer_ = node_.create_wall_timer(
```

Comment:
I'd recommend a ROS time timer by default, unless there's a specific need for system time. There's a [free function `rclcpp::create_timer()` to which a ROS time clock](https://github.com/ros2/rclcpp/blob/7c6785176a878cf16eb98995b732f405b9e776ed/rclcpp/include/rclcpp/create_timer.hpp#L107) can be passed.

Reply:
I'll change the 4 instances to use the new [`create_timer`](https://docs.ros.org/en/rolling/p/rclcpp/generated/classrclcpp_1_1Node.html?highlight=node#_CPPv4I000EN6rclcpp4Node12create_timerEN6rclcpp12GenericTimerI9CallbackTE9SharedPtrENSt6chrono8durationI12DurationRepT9DurationTEE9CallbackTN6rclcpp13CallbackGroup9SharedPtrE) method

Reply:
https://github.com/locusrobotics/fuse/pull/283/commits/59d10ddc448f2a21928d5104dabe494e0e37f8e4